nodejs/src/uws.js

```diff
@@ -437,18 +443,16 @@ class Server extends EventEmitter {
             _upgradeReq = null;
         });
 
-        if (options.port) {
-            if (options.host) {
-                this.httpServer.listen(options.port, options.host, ()=> {
-                      this.emit('listening');
-                      callback && callback();
-                });
-            } else {
-                this.httpServer.listen(options.port,  ()=> {
-                      this.emit('listening');
-                      callback && callback();
-                });
-            }
+        if (options.host) {
+            this.httpServer.listen(options.port, options.host, ()=> {
```

Comment:
Done. There's also other style issues like too long lines (some are more than 120 characters long). Don't know what's your style. I can fix that if you'd like me to.